<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileOrientedDownloader</class>
 <widget class="QDialog" name="FileOrientedDownloader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RSIG3D File Oriented Downloader</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>105</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QToolButton" name="fileDownloaderDirButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>72</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>72</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>72</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>Directory:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="fileDownloaderDirLineEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>441</width>
        <height>22</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>22</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1024</width>
        <height>22</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>371</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;b&gt;Files will be downloaded to&lt;/b&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Available disk space (GB):</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="fileDownloaderAvailableSpace">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>56</y>
        <width>64</width>
        <height>24</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="numDigits">
       <number>4</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Required disk space (GB):</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="fileDownloaderRequiredSpace">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>80</y>
        <width>64</width>
        <height>24</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="numDigits">
       <number>4</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>&lt;b&gt;Available data products:&lt;/b&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="fileDownloaderListWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="fileDownloaderStatus">
           <property name="text">
            <string>progress</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="fileDownloaderProgressBar">
           <property name="value">
            <number>0</number>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>39</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="fileDownloaderDownloadButton">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Download</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fileDownloaderCancelButton">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>false</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
